### DISCUSSION TITLE
My AI chatbot says that I've used up my message limit.

### DISCUSSION THREAD

**Discussion by mrodiculous** [AntlerVC/firetable#1425] [Support / Q&A] [answered]
Hi, I have a pro paid account and I keep getting the message, "Looks like you have used up your message limit! But don't worry, you can keep the conversation going by creating your own WhatsApp bot for free using your OpenAI account. Go to aibot.how to get started. It's super easy, without using any code, and your data stays secure on your own OpenAI account. Give it a try!"  What is wrong?

**Comment by Chadha93** [comment, marked as answer]
If your AI chatbot says that you've used up your message limit, it means that you have reached the maximum number of messages that you can send or receive within a specific time frame. This is usually set by the platform or service that you are using to host your chatbot.

You should try [BuildShip](https://buildship.app/) to create this workflow. 
Video - https://www.youtube.com/watch?v=QyR8Sautvhg
